### PR TITLE
donut chart for max calculator

### PR DESCRIPTION
**Pull request by Ilze-R**

### CODE REVIEW

File: src/app/donut/donut.component.ts [hunk 1]
```diff
@@ -17,9 +17,10 @@ const COLORS = ['#c5cae9', '#FFCB6F', '#7986cb', '#4FA57F', '#FE8A7F'];
   styleUrls: ['./donut.component.scss'],
 })
 export class DonutComponent implements OnInit, OnChanges {
-  @Input() mainLabel: string;
   @Input() data: number[];
   @Input() labels: string[];
+  @Input() maxData: any[];
```
Comment (kyskiz) [inline src/app/donut/donut.component.ts:22]: maybe we could use same 'data' binding?

File: src/app/donut/donut.component.ts [hunk 2]
```diff
@@ -90,7 +93,7 @@ export class DonutComponent implements OnInit, OnChanges {
         y: -30,
       },
       subtitle: {
-        text: 'montlhy payments',
+        text: data === this.data ? this.middleLabel[0] : this.middleLabel[1],
```
Comment (kyskiz) [inline src/app/donut/donut.component.ts:96]: maybe this could be passed as another binding from parent component, not to be hardcoded in 'donut'

Comment (kyskiz) [reply src/app/donut/donut.component.ts:96]: like @Input() middleLabel: text;

**Comment by kyskiz** [comment]
instead of   <app-donut [maxData]="chartData" 
just simply do 
<app-donut [data]="chartData" 
  
  I believe both types are number[] in practice
  in this way chart reusability is improved, we use same component, just adjust the parameters we provide to chart component

**Review by swoogie — APPROVED** [review]
legit

**Pull request merged**